### ISSUE TITLE
make jbang --fetchdeps=pom.xml myfile.java to get //deps

### ISSUE THREAD

**Issue by maxandersen** [maxandersen/jbang#43]

**Comment by maxandersen** [comment]
considered parsing xml directly but that wouldn't resolve properties so better to fully do it.

can be done with #44  using shrinkwrap or rely on maven doing `mvn dependency:list -DincludeScope=runtime -DexcludeTransitive -DappendOutput=true -DoutputFile=/Users/max/test.txt` to get dependencies.

**Comment by quintesse** [comment]
Didn't you implement this one already? Or is it different than the `--addeps` option?

**Comment by maxandersen** [comment]
no, not implemented yet - `--addeps` option was committed wrongfully.
